README.md

```diff
@@ -35,6 +34,14 @@ resource "flux_bootstrap_git" "this" {
 
 ## Guides
 
-* [Customize Flux configuration](https://registry.terraform.io/providers/fluxcd/flux/latest/docs/resources/bootstrap_git#customizing-flux)
-* [Bootstrap with GitHub](https://registry.terraform.io/providers/fluxcd/flux/latest/docs/guides/github)
-* [Bootstrap with GitLab](https://registry.terraform.io/providers/fluxcd/flux/latest/docs/guides/gitlab)
+The following guides are available to help you use the provider:
+
+- [Configuration using a Github repository via SSH](examples/github-via-ssh)
+- [Configuration using a Github repository via SSH and GPG](examples/github-via-ssh-with-gpg)
+- [Configuration using a Github repository via SSH with flux customizations](examples/github-with-customizations)
+- [Configuration using a Github repository via SSH and GPG with inline flux customizations](examples/github-with-inline-customizations)
```

Comment:
Should be `Flux`, the lowercase is the CLI name. Please make this change in all example docs.

Reply:
Changes made.